### DISCUSSION TITLE
Certificate Issuing

### DISCUSSION THREAD

**Discussion by Ibrahim-Halil-Kuray** [ProxeusApp/community#25] [Ideas]
https://may4basel.dribdat.cc/project/2 
### Some observations and feedback from our Proxeus presentation to competent people who organize Hackathons in Switzerland.

Based on the feedback from participants and my own observations, it seems that everyone agrees that participants should receive a certificate for attending the hackathon. Although, they may not fully understand the section on blockchain technology, I believe they partially comprehend the functionality provided by Proxeus. It appears that using the testnet network may have made some people less serious about the project, but using real Eth may raise concerns about Gas Fees, especially since they are not fixed. This uncertainty around Gas Fees could make some people hesitant to get involved in the project.

"There are also some unclear issues regarding certificate creation. Anyone who registers with Proxeus can create a fake certificate using the appropriate template. In relation to this, I mentioned the KYC procedure and document signing feature to them. They suggested that an option similar to the one in the visual would be useful for sharing certificates."

<img width="570" alt="1" src="https://user-images.githubusercontent.com/71791007/236830321-716a19b8-b0c5-460a-8870-748c09a14676.png">
<img width="497" alt="2" src="https://user-images.githubusercontent.com/71791007/236830329-3d2778c2-ceaf-40d2-a734-d1a9c06f4e74.png">

If there are things I know wrong or missing about these issues I mentioned, or if you have a suggestion, please do not hesitate to express your opinions.

**Comment by loleg** [comment]
Thanks for the feedback! It will be quite useful as we develop this use case further.
